src/aof.c

```diff
@@ -997,6 +997,29 @@ int startAppendOnly(void) {
     return C_OK;
 }
 
+void startAppendOnlyWithRetry(void) {
```

Comment:
can't we consolidate that with restartAOFAfterSYNC?
e.g. delete that old one, print one log message before calling the new one and letting it do the rest..

Reply:
Ok, added a commit, please check if I understood you correctly.